### PR TITLE
Stickers

### PR DESCRIPTION
**Pull request by jilaga**

### CODE REVIEW

**Comment by bebatut** [comment]
Thanks.

Few requests:
- Can you rename the folder to `stickers`?
- Can use only small case in the file names?

Thank you!

**Comment by jilaga** [comment]
done.

File: stickers/Ols!-transparent-bg.png
Comment (bebatut) [inline stickers/Ols!-transparent-bg.png:1]: Can you remove `!` and also rename `Ols` to `OLS`? Same for the other files

Comment (jilaga) [reply stickers/Ols!-transparent-bg.png:1]: How about renaming the image to ols-exclamation-mark?

Comment (bebatut) [reply stickers/Ols!-transparent-bg.png:1]: Perfect

**Pull request closed without merging**